src/averageAges.js

```diff
@@ -56,6 +90,33 @@ function calculateWomenAverageAge(people, withChildren) {
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+  const filterWomen = people.filter((person) => person.sex === 'f');
+
+  const mothers = [];
+  const ageDiffChildMother = [];
+  const ageDiffSonMother = [];
+
+  for (const key of filterWomen) {
+    people.some(child => {
```

Comment:
Why `some`? If you just iterating your array you should use forEach

Reply:
was trying to implement some or every  , my bad